development-docs/systemtests/io.strimzi.systemtest.cruisecontrol.CruiseControlST.md

```diff
@@ -33,6 +33,31 @@
 * [cruise-control](labels/cruise-control.md)
 
 
+## testAutoKafkaRebalanceScaleUpScaleDown
+
+**Description:** Test the Kafka Cruise Control auto-rebalance mechanism during scaling up and down of brokers.
+
+**Steps:**
+
+| Step | Action | Result |
+| - | - | - |
+| 1. | Create broker and controller KafkaNodePools. | Both KafkaNodePools are successfully created. |
+| 2. | Create KafkaRebalance templates for scale-up and scale-down operations. | KafkaRebalance templates are created and annotated as configuration templates. |
+| 3. | Deploy Kafka cluster with Cruise Control using the defined templates for auto-rebalance. | Kafka cluster with Cruise Control is deployed, configured with the specified auto-rebalance templates. |
+| 4. | Ensure the Kafka auto-rebalance status is Idle. | Kafka auto-rebalance status is confirmed to be Idle. |
+| 5. | Scale Kafka up to a higher number of brokers. | Kafka brokers are scaled up, and Cruise Control initiates rebalancing in ADD_BROKERS mode. |
+| 6. | Verify that Kafka auto-rebalance status transitions to RebalanceOnScaleUp and then back to Idle. | Auto-rebalance status moves to RebalanceOnScaleUp during scaling and returns to Idle after rebalancing completes. |
+| 7. | Check that topic replicas are moved to the new brokers. | Topic replicas are distributed onto the newly added brokers. |
+| 8. | Change number of replicas of Kafka cluster to initial replicas within KafkaNodePool (i.e., 3 brokers) | KafkaNodePool is set successfully to 3 replicas and auto-rebalance is triggered. |
+| 9. | After auto-rebalance is done, Kafka cluster is scale-down to the original number of brokers. | Kafka brokers are scaled down before of free partitions |
```

Comment:
"Kafka brokers are scaled down before of free partitions" ... this is an English thing on my side maybe, but I don't get what you mean, mostly the part "before of free partitions".

Reply:
Hmm, probably exhaustion from writing so much documentation...btw updated sounds better?
```
Kafka brokers are scaled down due to free partitions, and there are no hosted partitions anymore that would prevent such an operation.
```

Reply:
It would be more ...

```
Kafka brokers are scaled down because there are no hosted partitions anymore that would prevent such an operation.
```

Reply:
Updated